src/main/resources/application-dev.yaml

```diff
@@ -1,8 +1,14 @@
 no.nav.security.jwt:
-  issuer.azuread:
-    discoveryurl: https://login.microsoftonline.com/navq.onmicrosoft.com/v2.0/.well-known/openid-configuration
-    accepted_audience: ${CLIENT_ID}
-    cookie_name: azure_token
+  issuer:
+    selvbetjening:
+      discoveryurl: http://metadata
+      accepted_audience: aud-localhost
+      cookie_name: localhost-idtoken
+    # Kommenteres ut ved lokal testing av innsending av søknad
```

Comment:
Brukes ikke dette bare lokalt siden det er i -dev filen? Isåfall er kommentaren unødvendig.

Hvis ikke så burde vil det som er kommentert ut kommenteres inn igjen før dere merger

Reply:
Må være sånn for å teste innsending, men motsatt hvis man skal teste saksbehandler, men gjør ikke vi.